easybuild/easyblocks/n/numpy.py

```diff
@@ -204,6 +204,17 @@ def get_libs_for_mkl(varname):
             'includes': includes,
         }
 
+        numpy_version = LooseVersion(self.version)
+
+        if numpy_version < '1.26':
```

Comment:
```suggestion
        if LooseVersion(self.version) < LooseVersion('1.26'):
```

Just as a suggestion. This would be similar to all the checks below.

Reply:
Yes I intended to replace those too but forgot. Anyway enough for now